docs/_layouts/post.html

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+---
+<link rel="stylesheet" href="{{site.baseurl}}/css/pygments.css">
+<main>
+  <nav class="toc">
+    <ul data-depth="1">
+
+      <li data-level="H2">
```

Comment:
not sure whether this data-level is needed anymore

Reply:
Yeah I think it's not needed anymore. Also remove `data-depth` too.